src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppresswithnearbyplaintextcommentfilter/InputSuppressWithNearbyPlainTextCommentFilterCommentFormat.css

```diff
@@ -0,0 +1,96 @@
+/*
+SuppressWithNearbyPlainTextCommentFilter
+commentFormat = cs (\\w+)
+checkFormat = (default).*
+messageFormat = (default)(null)
+idFormat = (default)(null)
+influenceFormat = (default)0
+
+com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck
+fileExtensions = (default)all files
+ignorePattern = (default)^(package|import) .*
+max = 90
+
+com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineCheck
+format = this should not appear
+message = (default)(null)
+ignoreCase = (default)false
+minimum = (default)0
+maximum = (default)0
+ignoreComments = (default)false
+
+
+*/
+
+.div-1 main {
+  display: grid;
+  grid-template-rows: auto auto 1fr;
+  grid-template-columns: auto auto 1fr auto;
```

Comment:
Remove extra lines, that doesn't contribute to structural diversity.

Reply:
Done